etc/kayobe/pulp-repo-versions.yml

```diff
@@ -16,7 +16,8 @@ stackhpc_pulp_repo_centos_stream_9_nfv_openvswitch_version: 20230929T005202
 stackhpc_pulp_repo_centos_stream_9_openstack_yoga_version: 20231005T010906
 stackhpc_pulp_repo_centos_stream_9_opstools_version: 20230615T071742
 stackhpc_pulp_repo_centos_stream_9_storage_ceph_pacific_version: 20230709T010022
-stackhpc_pulp_repo_docker_ce_ubuntu_version: 20231020T014922
+stackhpc_pulp_repo_docker_ce_ubuntu_focal_version: 20231020T014922
+stackhpc_pulp_repo_docker_ce_ubuntu_jammy_version: 20231020T014922
```

Comment:
These are probably wrong

Reply:
https://ark.stackhpc.com/pulp/content/docker-ce/ubuntu-focal/
https://ark.stackhpc.com/pulp/content/docker-ce/ubuntu-jammy/

Reply:
Latest are
- [20240711T233948/](https://ark.stackhpc.com/pulp/content/docker-ce/ubuntu-focal/20240711T233948/)
- [20240711T233948/](https://ark.stackhpc.com/pulp/content/docker-ce/ubuntu-jammy/20240711T233948/)

Reply:
Latest don't work because they are missing dependencies from other repos which are behind, so we'd need to do a full refresh. Any idea what we should do?

Reply:
looks better now